sdk/eventhub/azure-eventhub/azure/eventhub/_connection_manager.py

```diff
@@ -112,5 +55,5 @@ def get_connection_manager(**kwargs):
     # type: (...) -> 'ConnectionManager'
     connection_mode = kwargs.get("connection_mode", _ConnectionMode.SeparateConnection)
     if connection_mode == _ConnectionMode.ShareConnection:
-        return _SharedConnectionManager(**kwargs)
+        pass
```

Comment:
nit: I think we could remove those all since we always return `_SeparateConnectionManager`